bee-protocol/src/tangle/metadata.rs

```diff
@@ -78,6 +88,29 @@ impl TransactionMetadata {
         self.solidification_timestamp = timestamp;
     }
 
+    pub fn cone_index(&self) -> Option<MilestoneIndex> {
+        self.cone_index
+    }
+
+    pub fn set_cone_index(&mut self, cone_index: MilestoneIndex) {
+        self.cone_index = Some(cone_index);
+    }
+
+    pub fn otrsi(&self) -> Option<MilestoneIndex> {
+        self.otrsi
+    }
+
+    pub fn set_otrsi(&mut self, otrsi: MilestoneIndex) {
+        self.otrsi = Some(otrsi);
+    }
+
+    pub fn ytrsi(&self) -> Option<MilestoneIndex> {
+        self.ytrsi
+    }
+
+    pub fn set_ytrsi(&mut self, ytrsi: MilestoneIndex) {
+        self.ytrsi = Some(ytrsi);
+    }
```

Comment:
sorry, but I can't sleep if you don't put a blank line here :lollipop:

Reply:
i guess i need to get some glasses 💯